### PR TITLE
[NLP-175] html email template

### PR DESCRIPTION
**Pull request by dhyxy**

### CODE REVIEW

File: server/risk_scores/risk_assessment.py [hunk 1]
```diff
@@ -72,28 +72,33 @@ def get_incident_similarity(prev_incident: submit_schema.Form, current_incident:
     return coefficient * similarity
 
 
-def get_incident_recency(prev_incident: submit_schema.Form, current_incident: submit_schema.Form, timeframe):
+def get_incident_recency(prev_incident: submit_schema.Form,
```
Comment (JCayabyab) [inline server/risk_scores/risk_assessment.py:75]: These autoformatting changes might result in merge conflicts with some incoming PRs.

File: server/risk_scores/risk_assessment.py [hunk 2]
```diff
@@ -126,23 +131,38 @@ def get_current_risk_score(form: submit_schema.Form):
     return percent_of_max
 
 
+required_email_form_fields = {'staff_name', 'client_primary'}
+
+
 def get_risk_assessment(form: submit_schema.Form, timeframe) -> RiskAssessment:
     total_risk_score = get_current_risk_score(form)
-    + get_previous_risk_score(form, timeframe)
+    +get_previous_risk_score(form, timeframe)
```
Comment (JCayabyab) [inline server/risk_scores/risk_assessment.py:139]: Is this really how we format our stuff?

File: server/risk_scores/risk_assessment.py [hunk 3]
```diff
@@ -126,23 +131,38 @@ def get_current_risk_score(form: submit_schema.Form):
...
+            email_dict = {
+                k: v
+                for k, v in form.dict().items()
+                if k in required_email_form_fields
+            }
+            email_dict.update({
+                'risk_score': assessment.value,
+                'risk_score_raw': total_risk_score
+            })
+            email_high_risk_alert(email_dict)
```
Comment (JCayabyab) [inline server/risk_scores/risk_assessment.py:152]: You removed the if-statement, this sends no matter what

File: server/risk_scores/risk_assessment.py [hunk 4]
```diff
@@ -126,23 +131,38 @@ def get_current_risk_score(form: submit_schema.Form):
...
             return assessment
     else:
         return RiskAssessment.UNDEFINED
 
 
-def email_high_risk_alert(form_values: dict):
-    form_values = (
-        f"<b>{field}</b>: {value}" for field, value in form_values.items())
+html_template_filename = "/.template.html"
```
Comment (JCayabyab) [inline server/risk_scores/risk_assessment.py:158]: Is there a reason why we start the filename with the `.`?

File: server/risk_scores/.template.html [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<h3 style="color:black">Risk Score Report</h3>
+<p style="color:black">A recent CIR that was filled out by <b>{staff_name}</b> regarding client <b>{client_primary}</b> reported a risk
+    score of {risk_score} [{risk_score_raw:.3f}].</p>
```
Comment (JCayabyab) [inline server/risk_scores/.template.html:3]: End files with newlines

**Review by JCayabyab — CHANGES_REQUESTED** [review]
In addition to the other requested changes, let's undo the rest of the formatting changes in the `risk_assessment.py` file to avoid the headache of having to deal with the merge conflicts with upcoming PRs.

File: server/risk_scores/risk_assessment.py (hunk 2 above)
Comment (dhyxy) [reply server/risk_scores/risk_assessment.py:139]: yapf is weird

File: server/risk_scores/risk_assessment.py (hunk 4 above)
Comment (dhyxy) [reply server/risk_scores/risk_assessment.py:158]: just thought it looks cool ngl, lemme fix

**Review by JCayabyab — APPROVED** [review]

**Pull request merged**